### DISCUSSION TITLE
Why can't I access page.github.com, and can't open your own git pages repository?

### DISCUSSION THREAD

**Discussion by fu1996** [community/community#31300] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/42232973/187344223-484cbac2-ae3c-4f39-aec4-40e83846526d.png)
I created the GIT page according to the official document, and suddenly found that the managed warehouse could not be opened today, but the interface was still accessible.

repo:https://github.com/fu1996/fu1996.github.io
blog:https://fu1996.github.io/

**Comment by airtower-luna** [comment]
Looks to me like it's working, so maybe it was a temporary problem?

**Reply by fu1996** [reply]
The blog is normally accessible, but the repository hosting the blog is inaccessible. Opening is 404. The problem I want to solve now is: open the repository hosting this blog normally: https://github.com/fu1996/fu1996.github.io 
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/42232973/189480061-cecb8d38-da2d-4866-9d6f-ec5cd84ab40a.png">

**Reply by fu1996** [reply]
<img width="1199" alt="image" src="https://user-images.githubusercontent.com/42232973/189480097-ed0f0250-4c70-4c58-97d5-6945645b1872.png">

**Comment by jfullstackdev** [comment]
and seems there are no errors at the Developer Tools anymore

![image](https://user-images.githubusercontent.com/47092464/187467985-a66d8847-b411-47c1-8323-8b697e7d4a39.png)

**Reply by fu1996** [reply]
The blog is normally accessible, but the repository hosting the blog is inaccessible. Opening is 404. The problem I want to solve now is: open the repository hosting this blog normally: https://github.com/fu1996/fu1996.github.io
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/42232973/189480052-fc86189a-9a66-48fe-a99b-8c2bc08fdd1d.png">

**Comment by fu1996** [comment, marked as answer]
It seems to be a DNS problem. I added GitHub DNS to solve this problem. thank you very much。
